gtsfm/evaluation/metrics.py

```diff
@@ -167,6 +167,9 @@ def _create_summary(self, data: np.ndarray) -> Dict[str, Any]:
         """
         if data.ndim != 1:
             raise ValueError("Metric must be a 1D distribution to get summary.")
+        if data.size == 0:
```

Comment:
thanks for the PR, Ayush. In what scenario does this situation occur?

Reply:
have we observed this occurring before?

Reply:
This is coming from the metric of average reprojection error on rejected tracks. Sometimes, no tracks are rejected and hence cannot compute summary statistics.

Reply:
i see, thanks